nni/tools/nnictl/launcher.py

```diff
@@ -592,16 +592,21 @@ def create_experiment(args):
         print_error('Please set correct config path!')
         exit(1)
     experiment_config = get_yml_content(config_path)
-    try:
-        config = ExperimentConfig(**experiment_config)
-        experiment_config = convert.to_v1_yaml(config)
-    except Exception:
-        pass
+    
     try:
         validate_all_content(experiment_config, config_path)
-    except Exception as e:
-        print_error(e)
-        exit(1)
+    except Exception:
+        print_warning('Validation with V1 schema failed. Trying to convert from V2 format...')
```

Comment:
When use V2 yaml file, there always be a warning here?

Reply:
yes, this is a content that I merge from yuge's pr. I think it is okay for now